packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -916,6 +918,50 @@ export const validateAndParseAddress = (
   return _validateAndParseAddressFn(address);
 };
 
+export async function getEstimatedFees(
```

Comment:
add some comment on the function, and may be highlight it is only for Cairo1 deploy